### ISSUE TITLE
Oxygen tank multiplication

### ISSUE THREAD

**Issue by WarAndroid** [Meldexun/BetterDiving#104]
**Describe the bug**
This bug gives you a copy of any oxygen tank, the tank will have the same max capacity as the exact capacity it was left at during the glitch (so if a small tank will have 9 seconds left you will have 1 tank with 30 seconds max and 1 glitched one with 9 seconds max)

**To Reproduce**
1. Fill any oxygen tank with oxygen.
2. Equip it with a mask and hop into the water.
3. Open your inventory.
4. Try to time shift clicking your oxygen tank with the air decrease rate.
5. Repeat step 4 until you have 2 oxygen tanks

**Versions**
Better Diving: BetterDiving-1.16.5-1.2.3
Forge: 1.16.5 - 36.1.0
Minecraft: 1.16.5

This bug was tested as the only forge mod

**Comment by Meldexun** [comment]
I tested it and it seems to only be a client-server synchronization issue. That means that the duplicate item is only present on the client and not on the server. Thus when clicking the duplicate item it disappears.
This makes this issue not high priority.
Also I don't know if I can fix this issue as this sounds more like a vanilla problem.

**Comment by WarAndroid** [comment]
Have you tried shift clicking it rather than dragging and dropping?, for me it works that way

**Comment by Meldexun** [comment]
I was shift clicking and I was able to reproduce the issue. But for me the duplicate item vanishes when clicking it.
That means for me it's only a minor visual/synchronization glitch and not an item duplication bug.

**Comment by WarAndroid** [comment]
Tho Oxygen drains normally as long as you only shift click thus massively increasing the amount of time you can spend in water, which is almost the same as duplication as long as you have original item somewhere

**Comment by Meldexun** [comment]
I don't understand what you mean. But when the item is not present on the server then it can't be used to "boost" your oxygen.

**Comment by WarAndroid** [comment]
I tried it in Singleplayer if this is what you mean, it works for me

**Comment by Meldexun** [comment]
No, there is always a client and a server. In singleplayer it is called an integrated server otherwise it would be a dedicated server.
